modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -0,0 +1,52 @@
+= Defining a Custom Ingress Configuration
+
+[NOTE]
+====
+The tasks outline in this topic must be performed by an IT administrator
+====
+
+Runtime Fabric on Self-managed Kubernetes enables you to edit a template file that specifies a custom Ingress configuration. Using this file you can define annotations, rewrite rules, the ingressClassName, and more.
+
+The template file is a Kubernetes ConfigMap file. After customizing the template, you can use the `rtfctl apply` command-line utility to apply the custom Ingress configuration to your Kubernetes cluster.
+
+If you do not apply a custom template file, Runtime Fabric uses a default configuration that works with the out-of-the-box Kubernetes Ingress configuration.
```

Comment:
This may seem slightly confusing. What we really we want to communicate to our customers is:
1. We (RTF) tries to create a kubernetes default ingress resource if no configmap data is applied/configured.
2. The default ingress resource may/may not work depending on the app-path and the ingress controller being used by the customer. For e.g. if the app endpoint expects subpaths in the request, and the customer is using nginx ingress controller, the default ingress resource we create will not _"work"_  out of the box when the API endpoint is hit.

Reply:
We can also perhaps consider providing a link to Kubernetes documentation for default ingressResource that we implement/create in our software by default.

Reply:
Agree w/ Karan. Instead, we can clarify the following:
```If you do not apply a custom template file, Runtime Fabric uses a default configuration that may not work with the chosen ingress controller```

Reply:
@sarjeet2013 I think that's too vague. We should at least tell them what to do if it doesn't work, which I assume is to create a custom configuration?

Reply:
- RTF tries to create a default ingress resource definition.
- Depending on your ingress controller, this resource definition may not work.
- In this case, you must apply a custom ConfigMap using the template.